### DISCUSSION TITLE
Check code if its ai written

### DISCUSSION THREAD

**Discussion by ghost** [jplag/JPlag#1932] [Q&A]
I am new to this programming sector. I have assignment and i took help from tutorials and sites for my assignment. My professor sent me this github repo when i asked him to check my code for plagiarism or ai detection. So, can you help me to guide on how to use this. I tried uploading py file into given link but it doesn't work.

**Comment by tsaglam** [comment]
You cannot use JPlag to check a single program for AI usage. What you can do is compare multiple programs with each other. Here, it does not matter what the origin (ai/human/etc.) of the programs is; JPlag will check the structural similarity. You could compare your program with the code bases from which you took code snippets. If you want to do that, refer to the readme regarding usage. However, educators typically use JPlag on a course dataset of multiple student programs.
